tests/functional/relation_tests/dynamic_table_tests/models.py

```diff
@@ -48,3 +62,31 @@
 ) }}
 select * from {{ ref('my_seed') }}
 """
+
+
+DYNAMIC_ICEBERG_TABLE_ALTER = """
+{{ config(
+    materialized='dynamic_table',
+    snowflake_warehouse='DBT_TESTING',
+    target_lag='5 minutes',
+    refresh_mode='INCREMENTAL',
+    table_format="iceberg",
+    external_volume="s3_iceberg_snow",
```

Comment:
Can we add a test to validate changing the external volume works?

Reply:
(realizing that means we need to setup a second external volume)

Reply:
We opted against this for now!